Dockerfile

```diff
@@ -1,46 +1,9 @@
-FROM golang:1.19.5-alpine3.17 AS go-builder
-# arch can be either x86_64 for amd64 or aarch64 for arm
-ARG arch=x86_64
-ARG libwasmvm_version=v1.1.1
-ARG libwasmvm_aarch64_sha=9ecb037336bd56076573dc18c26631a9d2099a7f2b40dc04b6cae31ffb4c8f9a
-ARG libwasmvm_amd64_sha=6e4de7ba9bad4ae9679c7f9ecf7e283dd0160e71567c6a7be6ae47c81ebe7f3
+FROM scratch
 
-# this comes from standard alpine nightly file
-#  https://github.com/rust-lang/docker-rust-nightly/blob/master/alpine3.12/Dockerfile
-# with some changes to support our toolchain, etc
-RUN set -eux; apk add --no-cache ca-certificates build-base;
```

Comment:
Don't we need CA certs in scratch image? Maybe better to use distroless image like `gcr.io/distroless/static-debian11:nonroot` which has  CA certs included?

Reply:
I'm not sure if archwayd is doing any calls requiring CA certs, maybe it is not needed

Reply:
@aelesbao @fdymylja @spoo-bar do you guys have insights on this?

Reply:
unfortunately no:(

Reply:
@kayano @jlehtimaki `archwayd` makes HTTPS calls when it's used as an RPC client, like this:

```sh
docker run --rm -it archwaynetwork/archwayd tx bank send --chain-id constantine-2 --node 'https://rpc.constantine-2.archway.tech:443' ...
```

The current version of the `archway-cli` [depends on this](https://github.com/archway-network/archway-cli/blob/main/src/clients/archwayd/index.js#L123-L130).

Reply:
ok, I think this was for the wget part in the dockerfile...

Reply:
If removing the certificates doesn't affect this kind of RPC request, then we're good to go

Reply:
@aelesbao we are terminating SSL on the LB/Ingress so the actual binary should not use SSL/certs.
I'll test this never the less..